test/verify/check-testlib

```diff
@@ -174,12 +174,12 @@ class TestRunTest(MachineCase):
         process = subprocess.run([run_tests, "--test-dir", VERIFY_DIR, "--list", "TestHostEditing.testLimits",
                                   "TestHostSwitching.testEdit"], env=env, capture_output=True)
         stdout = process.stdout
-        self.assertRegex(stdout, b"\nTestHostEditing.testLimits # RETRY 1 \(test affected tests 3 times\)\n")
-        self.assertRegex(stdout, b"\nTestHostEditing.testLimits # RETRY 2 \(test affected tests 3 times\)\n")
-        self.assertRegex(stdout, b"\nTestHostEditing.testLimits\n")
-        self.assertNotRegex(stdout, b"RETRY 3")
-        self.assertRegex(stdout, b"\nTestHostSwitching.testEdit\n")
-        self.assertNotRegex(stdout, b"TestHostSwitching.testEdit # RETRY")
+        self.assertIn(stdout, b"\nTestHostEditing.testLimits # RETRY 1 (test affected tests 3 times)\n")
```

Comment:
Wrong argument order -- assertIn has parameters (needle, haystack), while assertRegex is the other way round.

I'm actually rather shocked that the test succeeded anyway -- how can that be??